aws/modules/consolidate-logs/module.ftl

```diff
@@ -0,0 +1,226 @@
+                                            "store": {
+                                                "Tier": "mgmt",
+                                                "Component": "baseline",
+                                                "DataBucket" : "opsdata",
+                                                "Instance" : "",
+                                                "Version": "",
+                                                "Role": "datafeed"
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "LogFilters": {
+                "all-logs": {
+                    "Pattern": ""
+                }
+            },
+            "LoggingProfiles" : {
+                "consolidate" : {
+                    "ForwardingRules": {
+                        "store": {
+                            "Filter": "all-logs",
+                            "Links": {
+                                "store": {
+                                    "Tier": "mgmt",
+                                    "Component": "baseline",
+                                    "Instance": "",
+                                    "Version": "",
+                                    "DataBucket" : "opsdata",
+                                    "Role" : "produce"
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "DeploymentProfiles" : {
+                deploymentProfile : {
+                    "Modes" : {
+                        "*" : {
+                            "*" : {
+                                "Profiles" : {
+                                    "Logging" : "consolidate"
+                                }
+                            },
+                            "apigateway" : {
+                                "AccessLogging" : {
+                                    "Enabled" : true,
+                                    "aws:KinesisFirehose" : true,
+                                    "aws:KeepLogGroup" : true
+                                },
+                                "WAF" : {
+                                    "Profiles" : {
+                                        "EnableLogging" : true,
+                                        "Logging" : "consolidate"
+                                    }
+                                }
+                            },
+                            "cdn" : {
+                                "EnableLogging" : true,
+                                "WAF" : {
+                                    "Profiles" : {
+                                        "EnableLogging" : true,
+                                        "Logging" : "consolidate"
```

Comment:
I thought WAF didn't need logging profiles any more?

Reply:
Yep you're right - I was going of the Schema that is on the hamlet.io site which needs to be setup with CI to update on its own. I should have known this though, given I removed them! 

I'll fix this one up now.